### DISCUSSION TITLE
Site brand link in header does not respect baseURL config

### DISCUSSION THREAD

**Discussion by chtsao** [reuixiy/hugo-theme-meme#406] [Q&A] [answered]
Thanks for creating meme theme. I have used it for years and just love it!

Just few days ago, I created a hugo site and served as a Gitlab page.  
Everything works well except that the title links to my Gitlab io (https://chtsao.gitlab.io/)
rather than the baseURL (https://chtsao.gitlab.io/i2p2022/)

What's wrong? 

Your kind help is deeply appreciated. 

Here is my config.toml (in part)

># Site Settings
>baseURL = "https://chtsao.gitlab.io/i2p2022/"
>title = "Intro to Probability 2022"

**Comment by reuixiy** [comment, marked as answer]
I think this should be a bug

**Reply by chtsao** [reply]
Thanks for the prompt reply. I will wait for the update then. Thanks!

**Reply by reuixiy** [reply]
This should be resolved now, see https://github.com/reuixiy/hugo-theme-meme/pull/408

**Reply by chtsao** [reply]
It works now! Thanks a lot.
